### PR TITLE
#6147 refactor footer

### LINKED ISSUE #6147: Footer reorganization
I would sort these links a bit as we are having hello.kodadot.xyz I would make a new one column 

Incentives
--- 
- Referral Program
- Artist Ambassador
- Ambassador Program

Marketplace
---
- Contribute 
- Documentation -> fix link to https://developers.kodadot.xyz
- Faq
- Tutorial

KodaDot
---
- about
- carees -> migrate to hello and remove page from codebase
- MerchShop
- Press Kit

image

let me know your thoughts

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #6147
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI
Screen Shot 2023-06-05 at 9 04 41 PM

### CODE REVIEW

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
Some link on the footer would redirect to incorrect url.

image

**Comment by prachi00** [comment]
> Some link on the footer would redirect to incorrect url.
> 
> image

oh forgot the external boolean, thanks, fixed it

File: components/TheFooter.vue [hunk 1]
```diff
@@ -151,6 +198,11 @@ const menuMarketplace: Menu[] = [
     url: 'https://hello.kodadot.xyz/be-part-of-kodadot/join-as-a-developer',
     external: true,
   },
+  {
+    name: $i18n.t('faq'),
```
Comment (Jarsen136) [inline components/TheFooter.vue:202]: please upper case `FAQ`

**Review by Jarsen136 — COMMENTED** [review]
otherwise lgtm

**Review by preschian — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
the subscribe button is not sizing correctly 
image
1082px and around that 


same on 850px 
image

- I would maybe switch to mobile footer here

**Comment by prachi00** [comment]
@prury it is already center aligned 
Screen Shot 2023-06-06 at 9 27 42 PM

**Comment by prachi00** [comment]
@exezbcz fixed the 1st one
for the 2nd one, it seems mobile view starts at 768px, we use buefy helper for that https://bulma.io/documentation/helpers/visibility-helpers/

**Comment by prachi00** [comment]
> > @prury it is already center aligned Screen Shot 2023-06-06 at 9 27 42 PM
> 
> i mean like that: image

Screen Shot 2023-06-07 at 9 08 05 PM
done

**Comment by vikiival** [comment]
@exezbcz can we merge?

**Comment by yangwao** [comment]
> Documentation -> fix link to [developers.kodadot.xyz](https://developers.kodadot.xyz/)
Documetation should be renamed to "Developers" and point to -> developers.kodadot.xyz

**Comment by prachi00** [comment]
Screen Shot 2023-06-08 at 3 42 39 PM
done @yangwao

**Comment by prachi00** [comment]
@yangwao @vikiival  can we merge this

**Comment by vikiival** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.54 USD/DOT ~ 6.608 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x7484736c3840c3202a5fcab71966fd5d7f4339a0460fcbc722244ef7cf51910e](https://polkadot.subscan.io/extrinsic/0x7484736c3840c3202a5fcab71966fd5d7f4339a0460fcbc722244ef7cf51910e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**